app/scripts/controllers/preferences.js

```diff
@@ -32,7 +32,9 @@ class PreferencesController {
       tokens: [],
       suggestedTokens: {},
       useBlockie: false,
-      featureFlags: {},
+      featureFlags: {
+        betaUI: true,
```

Comment:
kind of ironic to push people toward the new UI and still call it betaUI here

Reply:
What do you mean?

Reply:
the flag is called "betaUI" and is references throughout the pull request.

when the reality is that it will become the production UI, by nature of being the primary one people get to see and is the natural progression of it.

the idea being that some OTHER unforeseen design become the new betaUI, or the community itself is leveraged to provide skins for metamask

Reply:
I don't follow, sorry. This is a feature flag that is now permanently enabled as the first step to us removing its usages. It is called `betaUI` because that's what the feature it _was_. We will be removing it because, yes, it is becoming the default UI. I don't think that's ironic/unexpected, that's the progression of a feature flag.

If you're interested in community skins, feel free open up a new issue detailing those ideas!

Reply:
I get it, that's funny. With hindsight, we could have named this parameter something like "ui2" instead. It's like finding an old can of soap in your basement that says "new improved formula!". How new? How beta?

Not a big deal, we can just remove this flag when it's no longer a beta.